.github/workflows/push.yml

```diff
@@ -38,3 +38,21 @@ jobs:
 
        - name: Perform CodeQL Analysis
          uses: github/codeql-action/analyze@v1
+
+  Code-Coverage:
+    runs-on: ubuntu-latest
+    needs: CodeQL
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/setup-node@v1
+        with:
+          node-version: ${{ matrix.node-version }}
+      - run: yarn install
+      - run: yarn test --watchAll=false --coverage
+      - name: Upload coverage to Codecov
+        uses: codecov/codecov-action@v1   
+      - name: Report code coverage
```

Comment:
Remove everything from this line downwards. We don't want `VeryGoodOpenSource/very_good_coverage@v1.1.1` run again at this stage. Only `codecov/codecov-action@v1`

Reply:
Ok